doc/api/util.md

```diff
@@ -165,6 +165,30 @@ stream.on('data', (data) => {
 stream.write('It works!'); // Received data: "It works!"
 ```
 
+ES6 example using class and extends
```

Comment:
`class` and `extends` are keywords in JavaScript, so suggest highlighting these two words :-)